docs/databags.md

```diff
@@ -18,6 +18,7 @@ The Cluster Hash is part of a plaintext data bag item that defines a logical gro
 * `['master_uri']` - SplunkAPI URI of the cluster master (Required for servers connecting to managed clusters)
 * `['settings']` -  Hash of Cluster settings (Required for servers connecting to managed clusters),
 * `['settings'][???]` - Valid values are those under the clustering stanza of [server.conf][]
+* `['settings'][???]['_cerner_splunk_indexer_count']` - The number of indexers to use for calculating maxTotalDataSizeMb for each index in combination with _maxDailyDataSizeMB in the index configuration.
```

Comment:
Fixed in https://github.com/cerner/cerner_splunk/pull/123/files#diff-a8cd9604d38ad43b8b92ae521a075a6aR21